src/Widgets/TestimonialSliderWidget/TestimonialSliderWidgetComponent.js

```diff
@@ -66,7 +66,11 @@ function sliderSettings(testimonials) {
     infinite: true,
     speed: 500,
     dotsClass: "testimonial-slider-widget--quote-portrait-wrapper",
-    customPaging: (i) => <button>{testimonialAuthorImages[i]}</button>,
+    customPaging: (i) => (
+      <button aria-label="Show testimonial">
```

Comment:
* [ ] We have a better option here: use the `authorName`. This way, a screen reader can tell the individual buttons apart.